e2e/plotting.spec.ts

```diff
@@ -68,7 +68,7 @@ test('plots a time vs shotnum graph and change the plot colour', async ({
   const chart = await popup.locator('#my-chart');
   const dimensions = await chart.boundingBox();
   expect(
-    await popup.screenshot({
+    await chart.screenshot({
```

Comment:
Hmm, I didn't know that we could take a screenshot of the element itself - this means we don't have to set the dimensions of the screenshot right? But I suppose if it's not broke then don't fix it!

Reply:
I took the dimensions out and it still worked fine! See commit https://github.com/ral-facilities/operationsgateway/pull/92/commits/2c3a0e4740123a8e2bdefc991c0e93f7e974bd75